packages/syft/setup.cfg

```diff
@@ -91,7 +91,7 @@ libs =
     openmined.psi
     pandas
     petlib
-    pillow>=8.1.2 #security-issues
+    pillow>=8.1.2,<=8.2.0 #security-issues
```

Comment:
I think this should be placed in dev or before ,initially 8.3.0 installed, on `ci-libs` when `pillow8.2.0` is download ,it should uninstall 8.3.0 ,there is an error at end in windows runner which prevents uninstallation  https://github.com/OpenMined/PySyft/runs/2985615595?check_suite_focus=true#step:7:557

Reply:
im seeing all the tests pass in CI